hack/latest-binaries.sh

```diff
@@ -4,23 +4,75 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
+
+BUCKET="amazon-eks"
```

Comment:
Why don't you just allow overriding this environment variable directly?
```suggestion
BINARY_BUCKET=${BINARY_BUCKET:-amazon-eks}
```

Reply:
Because the entrypoint into this project is `make` and one of the exposed variables is `binary_bucket_name`. I chose this approach to keep the bucket name consistent with this. 

```
make 1.28 binary_bucket_name=my-bucket
```
vs 

```
BINARY_BUCKET=my-bucket; make 1.28 binary_bucket_name=my_bucket
```

Unless I'm misunderstanding how 
`BINARY_BUCKET=${BINARY_BUCKET:-amazon-eks}` would work.

Reply:
You can `export` variables within the Makefile to child commands, so you could use `binary_bucket_region` and `binary_bucket_name` directly in the script as well.